hazelcast/src/test/java/com/hazelcast/cluster/ClusterShutdownTest.java

```diff
@@ -56,11 +57,19 @@ private void testClusterShutdownWithSingleMember(final ClusterState clusterState
         final Thread thread = new Thread(new Runnable() {
             @Override
             public void run() {
-                instances[0].getCluster().shutdown();
+                while (true) {
```

Comment:
Hmmm.. This looks strange to me. So it can happen that a shutdown call throws an exception and is ignored??